lang/main.json

```diff
@@ -864,6 +864,7 @@
         "header": "Help center"
     },
     "lobby": {
+        "knockingParticipantList" : "Knocking participant list",
```

Comment:
Where is this used? I don't see it in the code.

Reply:
I see it is not translated string from code.

Reply:
In fact it is part of a larger plan ^^ I make another pull request in order to replace the string by a var .. We are french and "Knocking" is just a joke for us 'knock, knock' :)

Reply:
Sorry, the "larger plan" was not deployed.. #7249